### PR TITLE
Snapshot disable spellcheck

### PR DESCRIPTION
**Pull request by AndreClaassen1**
I propose a new option --disable_spellcheck_on_simulator to disabling spell checking and auto correcting on the iOS simulator. I'm currently working on a journal app for Evernote and I need to demonstrate text input. Unfortunately, the iOS spell checker makes every screen shot nasty. So I make this proposal to disabling the spellchecking while making screenshots with snapshot. 

iphone4s--notutorial-01startscreen original

### CODE REVIEW

**Comment by KrauseFx** [comment]
Hey @Chaosspeeder, thanks for the pull request. I'm not 100% sure if this should be included to snapshot by default, since it's a very specific use case. Instead I'd love to find a way to allow `snapshot` to have those kind of extensions, without having to bundle them all in the actual `snapshot` gem.

**Comment by Chaosspeeder** [comment]
An extension system is a nice idea. For me, the problem is more common and
not so special. iOS spell checking uses unfortunately the system language
and not the app language. But either way, default or extension, would be a
good way to deal with it.

Thank you for your attention.

Best regards, Andre
Felix Krause <notifications@github.com> schrieb am So., 10. Apr. 2016 um
12:21:

File: snapshot/lib/snapshot/options.rb [hunk 1]
```diff
@@ -82,6 +82,11 @@ def self.available_options
                                      description: "Enabling this option will automatically erase the simulator before running the application",
                                      default_value: false,
                                      is_string: false),
+        FastlaneCore::ConfigItem.new(key: :disable_spellcheck_on_simulator,
```
Comment (KrauseFx) [inline snapshot/lib/snapshot/options.rb]: Negative booleans are not ideal, could you switch it around and call it `enable...`

File: snapshot/lib/snapshot/runner.rb [hunk 2]
```diff
@@ -136,6 +136,10 @@ def launch(language, locale, device_type, launch_arguments)
         uninstall_app(device_type)
       end
 
+      if Snapshot.config[:disable_spellcheck_on_simulator]
```
Comment (KrauseFx) [inline snapshot/lib/snapshot/runner.rb]: Move this check to the first line of `disable_spellcheck_on_simulator` instead

**Comment by KrauseFx** [comment]
Alright, thanks, good point. I added a comment, that needs to be addressed. Please squash the commits and let me know when it's ready :+1:

**Comment by Chaosspeeder** [comment]
HI @KrauseFx , 

this is my newest attempt. I switched to enable logic for my switch and tried to be confirm with apple in naming and wording of the code.

**Comment by st3fan** [comment]
@Chaosspeeder You can try to run your tests on my branch, https://github.com/fastlane/fastlane/pull/4035 which changes the device locale.

**Comment by Chaosspeeder** [comment]
@KrauseFx Please do not merge/push this pull request. I will first trying out #4313 which is a more general  solution to the problem.

**Comment by KrauseFx** [comment]
Thanks @Chaosspeeder, I'm closing this PR for now 👍

**Pull request closed without merging**